doc/source/whatsnew/v1.5.0.rst

```diff
@@ -435,6 +435,47 @@ Calling :meth:`.DataFrameGroupBy.value_counts` with ``observed=True`` would inco
     1  c    0
     dtype: int64
 
+.. _whatsnew_150.notable_bug_fixes.concat_unordered_categoricalindex_overrides_indices:
+
+Concatenating two ``CategoricalIndex`` variables with ``ordered=False`` yields correct result
```

Comment:
I don't think we need a whole entry for this bug fix. Just a 1 line description under the `Categorical` section.

Reply:
Hi @mroeschke, thanks for the feedback. I updated the PR.